src/explore-education-statistics-frontend/src/middleware/pages/__tests__/redirectPages.test.ts

```diff
@@ -299,33 +299,6 @@ describe('redirectPages', () => {
       nonRedirectedCases: findStatisticsReleasePageTestData.nonRedirectedCases,
     };
 
-  const dataTablesPublicationPageTestData: RoutePatternTestCases = {
-    routePattern: `data-tables/${publicationSlugPlaceholder}`,
-    redirectedCases: [
-      {
-        oldSlugNewSlugPairsByPlaceholder: {
-          [publicationSlugPlaceholder]: {
-            oldSlug: 'original-publication-slug-1',
-            newSlug: 'updated-publication-slug-1',
-          },
-        },
-      },
-    ],
-    nonRedirectedCases: [
-      {
-        oldSlugsByPlaceholder: {
-          [publicationSlugPlaceholder]: 'original-publication-slug-2',
-        },
-      },
-      // Matches redirect slugs from another redirect type
-      {
-        oldSlugsByPlaceholder: {
-          [publicationSlugPlaceholder]: 'original-methodology-slug-1',
-        },
-      },
-    ],
-  };
```

Comment:
I'm wondering if removing this is correct because this looks like it's covering the route data-tables/{publicationSlug} AFTER you've selected a publication in step 1 of the table tool.

Reply:
You're right, I've added this back in :)